modules/update-service-create-service-cli.adoc

```diff
@@ -0,0 +1,85 @@
+[id="update-service-create-service-cli_{context}"]
+= Creating an OpenShift Update Service application by using the CLI
+
+You can use the {product-title} CLI to create an OpenShift Update Service application.
+
+.Prerequisites
+
+* The OpenShift Update Service Operator has already been installed.
+
+.Procedure
+
+To create an OpenShift Update Service application by using the {product-title} CLI:
+
+. Configure the OpenShift Update Service target namespace, for example, `openshift-update-service`:
++
+[source,terminal]
+----
+$ NAMESPACE=openshift-update-service
+----
++
+The namespace must match the `targetNamespaces` value from the OperatorGroup.
+
+. Configure the name of the OpenShift Update Service application, for example, `service`:
++
+[source,terminal]
+----
+$ NAME=service
+----
+
+. Configure the local registry and repository for the release images, for example, `registry.example.com/openshift/release`:
++
+[source,terminal]
+----
+$ RELEASE_IMAGES=registry.example.com/openshift/release
+----
+
+. Set the local pullspec for the graph-data image to the graph-data container image created in xref:modules/update-service-graph-data.adoc#update-service-graph-data_{context}[Creating the OpenShift Update Service graph data container image], for example, `registry.example.com/openshift/graph-data:latest`:
++
+.Example output
+[source,terminal]
+----
+NAME                             AGE
+openshift-update-service-fprx2   4m41s
+----
++
+[source,terminal]
+----
+$ GRAPH_DATA_IMAGE=registry.example.com/openshift/graph-data:latest
+----
+
+. Create an OpenShift Update Service application object:
++
+[source,terminal]
+----
+$ oc -n "${NAMESPACE}" create -f - <<EOF
+apiVersion: updateservice.operator.openshift.io/v1
+kind: UpdateService
+metadata:
+  name: ${NAME}
+spec:
+  replicas: 2
+  releases: ${RELEASE_IMAGES}
+  graphDataImage: ${GRAPH_DATA_IMAGE}
+EOF
+----
+
+. Verify the OpenShift Update Service application:
+
+.. Use the following command to obtain a policy engine route:
++
+[source,terminal]
+----
+$ while sleep 1; do POLICY_ENGINE_GRAPH_URI="$(oc -n "${NAMESPACE}" get -o jsonpath='{.status.policyEngineURI}/api/upgrades_info/v1/graph{"\n"}' updateservice "${NAME}")"; SCHEME="${POLICY_ENGINE_GRAPH_URI%%:*}"; if test "${SCHEME}" = http -o "${SCHEME}" = https; then break; fi; done
+----
++
+You might need to poll until the command succeeds.
+
+.. Retrieve a graph from the policy engine:
++
+[source,terminal]
+----
+$ while sleep 10; do HTTP_CODE="$(curl --header Accept:application/json --output /dev/stderr --write-out "%{http_code}" "${POLICY_ENGINE_GRAPH_URI}?channel=stable-4.6")"; if test "${HTTP_CODE}" -eq 200; then break; fi; echo "${HTTP_CODE}"; done
```

Comment:
If channel is not always targets to stable-4.6, it would be better to have a variable for it.

Reply:
I assuming it's the version number within the channel spec you're referring to so I added more explanation and used the version variable.